content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -214,6 +290,23 @@ Apply the remote cluster configuration.
 $ istioctl --context ${REMOTE_CLUSTER_CTX} manifest apply -f istio-remote0-cluster.yaml
 {{< /text >}}
 
+Wait for the remote cluster to be ready.
+
+{{< text bash >}}
+$ kubectl --context=${REMOTE_CLUSTER_CTX} -n istio-system get pod
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-55f784779d-s5hwl   1/1     Running   0          91m
+istiod-7b4bfd7b4f-fwmks                 1/1     Running   0          91m
+prometheus-c6df65594-pdxc4              2/2     Running   0          91m
+{{< /text >}}
+
+{{< tip >}}
+The istiod deployment running in the remote cluster is providing automatic sidecar injection and CA
+services to the remote cluster's pods. These services were previously provided by the sidecar injector
+and Citadel deployments which no longer exist with Istiod. The remote cluster's pods are
```

Comment:
Add a comma after "Citadel deployments". Unless I'm misunderstanding the sentence, which I might be.

Reply:
```suggestion
and Citadel deployments, which no longer exist with Istiod. The remote cluster's pods are
```